core/src/test/scala/org/apache/spark/sql/delta/DeltaConfigSuite.scala

```diff
@@ -201,4 +202,40 @@ class DeltaConfigSuite extends SparkFunSuite
       assert(e.getMessage == msg)
     }
   }
```

Comment:
In this suite the test "we don't allow arbitrary delta-prefixed table properties" is failing because the error message needs to be changed, spark.databricks.delta.allowArbitraryProperties.enabled=true to spark.delta.allowArbitraryProperties.enabled=true